javascripts/buzz.js

```diff
@@ -116,7 +116,11 @@ app.buzz = {
             }
             d.updatedDate = jQuery.timeago(new Date(d.sys_created));
             d.sys_description = d.sys_description[0].substr(0,197) + '...';
-            d.permanentLink = "//planet.jboss.org/post/" + d.sys_content_id;
+            if (d.sys_url_view.startsWith('http://developerblog.redhat.com')) {
```

Comment:
@wesbos any ideas why this fails? I'm trying to use: https://github.com/redhat-developer/developers.redhat.com/blob/master/javascripts/extensions.js#L18, but I get the error: 'Uncaught TypeError: d.sys_url_view.startsWith is not a function'